docs/release.md

```diff
@@ -0,0 +1,98 @@
+## Release process
+
+To release a new version, you need to be a member of `gcp-variant-transforms`,
+and `gcp-variant-transforms-test` projects and have write access to the GitHub
+repo.
+
+The process is as follows:
+
+1. Navigate to
+   [Container Builder build history](https://console.cloud.google.com/gcr/builds?project=gcp-variant-transforms-test)
+   and verifly that the latest integraion tests have passed.
+1. Choose the appropriate version number by following the semantics in
+   https://semver.org/. See the **Summary** section for a quick overview.
+1. Update the version number in
+   [setup.py](https://github.com/googlegenomics/gcp-variant-transforms/blob/master/setup.py)
+   by submitting a change. Please note the commit SHA generated by GitHub (it's
+   a long string like `4ff1fdfb3957a115a80fe9343306952cd98b6e5c`). You may run
+   the following command to make the next steps easier:
+   `export COMMIT_SHA=<commit sha>`.
+1. Wait until the
+   [Cloud Build](https://console.cloud.google.com/gcr/builds?project=gcp-variant-transforms-test)
+   process builds a new image that contains the change submitted in the previous
+   step. The image has the form
+   `gcr.io/gcp-variant-transforms-test/gcp-variant-transforms:${COMMIT_SHA}`.
+1. The tests in the previous step only cover presubmit tests. We require the
+   release image to pass all tests including `huge_tests`. Run the following on
+   your workstation in a **clean** branch synced to the version change commit
```

Comment:
How do you do that? Could you add cmd for this too. :-)

Reply:
Done :)